tfs/src/main/resources/hudson/plugins/tfs/TeamPRPushTrigger/TeamPollingAction/index.jelly

```diff
@@ -0,0 +1,45 @@
+<!--
```

Comment:
What's the reason for this file?

Reply:
It's cloned from TeamPushTrigger. Turns out it's redudant for TeamPRPushTrigger -- it worked fine after removing it. So I've removed the whole file.